processing file asmFiles/msi_1.asm (1 of 5)
processing file asmFiles/msi_2.asm (2 of 5)
processing file asmFiles/msi_3.asm (3 of 5)
processing file asmFiles/msi_4.asm (4 of 5)
processing file asmFiles/msi_5.asm (5 of 5)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/msi_1.asm (1 of 7)
processing file asmFiles/dual.llsc.asm (1 of 1)
--- memsim.hex	2022-04-20 16:35:41.669277079 -0400
+++ memcpu.hex	2022-04-20 16:35:41.576273622 -0400
@@ -14,7 +14,9 @@
 :04000D00AC430008F8
 :04000E00FFFFFFFFF2
 :0400200000000002DA
+:0400220000000001D9
 :0400240000000002D6
+:0400260000000001D5
 :0400800034011FC068
 :0400810034020090B5
 :040082008C430000AB
processing file asmFiles/dual.llsc.asm (1 of 1)
--- memsim.hex	2022-04-20 16:43:11.619005013 -0400
+++ memcpu.hex	2022-04-20 16:35:41.576273622 -0400
@@ -14,7 +14,9 @@
 :04000D00AC430008F8
 :04000E00FFFFFFFFF2
 :0400200000000002DA
+:0400220000000001D9
 :0400240000000002D6
+:0400260000000001D5
 :0400800034011FC068
 :0400810034020090B5
 :040082008C430000AB
processing file asmFiles/dual.llsc.asm (1 of 1)
--- memsim.hex	2022-04-20 16:43:48.639381332 -0400
+++ memcpu.hex	2022-04-20 16:43:48.562378469 -0400
@@ -14,7 +14,9 @@
 :04000D00AC430008F8
 :04000E00FFFFFFFFF2
 :0400200000000002DA
+:0400220000000001D9
 :0400240000000002D6
+:0400260000000001D5
 :0400800034011FC068
 :0400810034020090B5
 :040082008C430000AB
